cypress/e2e/PageObjects/ProfilePO.cy.js

```diff
@@ -105,6 +105,36 @@ function ExpandCard(CardName) {
   })
 }
 
+function ClickonTaskLinks() {
+  return cy
+    .get("[data-cy='task-list']")
+    .find('a')
+    .each(($el1, index, $list) => {
+      cy.wrap($el1).click()
+      ExitBetaModal().should('be.visible')
+      StayOnBetabutton().click()
+      cy.wrap($el1).click()
+      ExitBetaModal().should('be.visible')
+      CloseModalButton().click()
+    })
+}
+
+function ExitBetaModal() {
+  return cy.get("[data-cy ='exitBetaModal']")
+}
+
+function StayOnBetabutton() {
+  return cy.get("[id ='modal-btn-close']")
```

Comment:
It would be good to keep it consistent and use `data-cy` where we can.

Reply:
I am going to work on adding tags and other code improvements for next few days